### PR TITLE
unify test case names

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

**Comment by LiliDeng** [comment]
@squirrelsc for ltp and xfstesting, what is the suggested prefix? `community`?

**Comment by squirrelsc** [comment]
> @squirrelsc for ltp and xfstesting, what is the suggested prefix? `community`?

How about keep `verify_`?

**Comment by LiliDeng** [comment]
@squirrelsc any comments? I will notify the guys before merging.

File: microsoft/testsuites/core/hv_module.py [hunk 1]
```diff
@@ -222,7 +222,9 @@ def verify_hyperv_modules(self, log: Logger, environment: Environment) -> None:
             min_core_count=4,
         ),
     )
-    def reload_hyperv_modules(self, case_name: str, log: Logger, node: Node) -> None:
+    def stress_reload_hyperv_modules(
```
Comment (squirrelsc) [inline microsoft/testsuites/core/hv_module.py]: It shouldn't be called stress. It's a P1 test case, and not slow.

Comment (LiliDeng) [reply microsoft/testsuites/core/hv_module.py]: updated

**Review by squirrelsc — DISMISSED** [review]
Please hold merging, until it's well communicated.

**Comment by mcgov** [comment]
Lili asked for a review, this shouldn't affect our workflows, LGTM

**Review by squirrelsc — DISMISSED** [review]

**Pull request merged**